### ISSUE TITLE
Building a destroyed pantheon app results in errr

### ISSUE THREAD

**Issue by pirog** [kalabox/kalabox#475]
replicate

```bash
kbox create pantheon 
cd appname
kbox destroy
export KALABOX_DEV=true # this is not essential to replicate
kbox build
kbox start
```

```js
EVENTS =>  Error dispatching event listeners [post-start-component].
Error: Non-zero exit code: 1 ln: failed to create symbolic link `/code/sites/default/files': No such file or directory

    at /usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:566:28
    at Object.result.wait (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:545:20)
    at /usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:636:18
    at Object.queryData (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:634:6)
    at Object.exports.queryData (/usr/local/bin/lib/engine.js.jx:637:27)
    at /Users/mpirog/Desktop/apps/florida/plugins/kalabox-plugin-pantheon/lib/env.js:351:28
    at /usr/local/bin/lib/core/events.js.jx:147:11
    at /usr/local/bin/lib/core/events.js.jx:146:24
    at /usr/local/bin/lib/core/events.js.jx:130:23
    at EventEngine._dispatch (/usr/local/bin/lib/core/events.js.jx:127:18)
    at /usr/local/bin/lib/core/events.js.jx:107:19
    at EventEngine.emit (/usr/local/bin/lib/core/events.js.jx:99:21)
    at Object.exports.emit (/usr/local/bin/lib/core/events.js.jx:230:4)
    at /usr/local/bin/lib/app.js.jx:467:24
    at startComponent (/usr/local/bin/lib/app.js.jx:466:4)
    at services.verify.then.then.then.map.concurrency (/usr/local/bin/lib/app.js.jx:509:12)
    at exports.start (/usr/local/bin/lib/app.js.jx:508:4)
    at /usr/local/bin/lib/app.js.jx:612:12
    at Object.exports.restart (/usr/local/bin/lib/app.js.jx:611:4)
    at Object.task.func (/usr/local/bin/plugins/kalabox-core/tasks/restart.js.jx:15:18)
    at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:857:15)
    at processTask (/usr/local/bin/bin/kbox.js.jx:166:16)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:215:12)
    at process._tickCallback (node.js:812:13)
    at main (/usr/local/bin/bin/kbox.js.jx:214:4)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:223:1)
    at Module._compile (module.js:518:26)
    at readX (module.js:1284:5)
    at Object.Module._extensions..jx (module.js:1711:10)
    at Module.load (module.js:345:36)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:377:17)
    at require (module.js:395:17)
    at Object.<anonymous> (_jx_source:17:3)
    at Module._compile (module.js:518:26)
    at Module.load (module.js:353:12)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:733:12)
    at startup (node.js:371:18)
    at node.js:1464:3
```

The reason for this is we've destroyed the data container and then on `kbox build` we aren't pulling the site again since that happens in post-create only. Possible resolutions

1. We only try to symlink if that dir exists (this is not a great idea)
2. We only symlink on the initial create
3. We also do a pullsite when we do on post-build (is this different than post install?)

**Comment by magbicaleman** [comment]
I'm on the same boat.

**Comment by pirog** [comment]
So i think the best resolution here is to have the "complete" destroy/build procedure be

```
kbox destroy
kbox build # recreates containers
kbox pull # recreates site data
```

Of course it makes way more sense to just do a `kbox rebuild` since this rebuilds your containers while preserving your app data, however that assume you are starting from a pre-defined app state. If you are doing app-plugin dev you will want to have the procedure above work.

**Comment by pirog** [comment]
Should be fixed in upcoming 0.10.2
